ui/packages/app/src/project_setting/form/Stream.js

```diff
@@ -21,10 +21,10 @@ export const Stream = ({
   const [streamError, setStreamError] = useState("");
 
   const onStreamChange = stream => {
-    let isValid = isDNS1123Label(stream);
+    let isValid = isValidK8sLabelValue(stream);
     if (!isValid) {
       setStreamError(
-        "Stream name is invalid. It should contain only lowercase alphanumeric and dash (-), and must start and end with an alphanumeric character"
+        "Stream name is invalid. It should contain only lowercase alphanumeric and dash (-), or underscore (_) or period (.)"
```

Comment:
Should we have `, and must start and end with an alphanumeric character"` added to the end of this string like for the error string in `Team.js`?

Reply:
fixed